types/forge-viewer/index.d.ts

```diff
@@ -344,11 +344,12 @@ declare namespace Autodesk {
           isGeomLeaf(): boolean;
           isMetadata(): boolean;
           isViewable(): boolean;
+          isViewPreset(): boolean;
```

Comment:
Is there any documentation for this (the package isn't on npm so I can't check the code). Same with `setView`? I was able to find `traverse` in the online docs.